vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSlideController.java

```diff
@@ -67,7 +67,7 @@ public String slide(Model model, @PathVariable("slideId") String slideId, @PathV
         if (module.getStartSequence() == null) {
             model.addAttribute("showAlert", true);
             model.addAttribute("message", "Sorry, module has not been configured yet.");
-            return "module";
```

Comment:
what happened to that page? It should still exist just in a different location?

Reply:
Then the module page needs to be fixed. I'm pretty certain that it was the same page before for the not configured message and the module. We don't want to have to change two pages if we make layout changes, if we can just change one instead.